MAINTAINERS.md

```diff
@@ -1,238 +1,82 @@
-% rm -rf zod/base/tests/
-% for desk in garden landscape bitcoin webterm; do \
-    rsync -avL --delete pkg/$desk/ zod/$desk/ \
-  done
-> |commit %base
-> |commit %garden
-> |commit %landscape
-> |commit %bitcoin
-> |commit %webterm
-> .multi/pill +solid %base %garden %landscape %bitcoin %webterm
-> .multi-brass/pill +brass %base %garden %landscape %bitcoin %webterm
-```
-
-And then of course:
+`develop` is the default branch in the repo, which means that all new pull
+requests target it by default. The general flow of a new feature or bug fix
+through the pipeline is:
 
 ```console
-> .solid/pill +solid
-> .brass/pill +brass
-> .ivory/pill +ivory
-```
-
-For an Urbit OS release, after all the merge commits, make a release with the
-commit message "release: urbit-os-v1.0.xx".  This commit should have up-to-date
-artifacts from pkg/interface and a new version number in the desk.docket-0 of
-any desk which changed.  If neither the pill nor the JS need to be updated (e.g
-if the pill was already updated in the previous merge commit), consider making
-the release commit with --allow-empty.
-
-If anything in `pkg/interface` has changed, ensure it has been built and
-deployed properly.  You'll want to do this before making a pill, since you want
-the pill to have the new files/hash.  For most things, it is sufficient to run
-`npm install; npm run build:prod` in `pkg/interface`.
-
-However, if you've made a change to Landscape's JS, then you will need to build
-a "glob" and upload it to bootstrap.urbit.org.  To do this, run `npm install;
-npm run build:prod` in `pkg/interface`, and add the resulting
-`pkg/arvo/app/landscape/index.[hash].js` to a fakezod at that path (or just create a
-new fakezod with `urbit -F zod -B bin/solid.pill -A pkg/arvo`).  Run
-`:glob|make`, and this will output a file in `fakezod/.urb/put/glob-0vXXX.glob`.
-
-Upload this file to bootstrap.urbit.org, and modify `+hash` at the top of
-`pkg/arvo/app/glob.hoon` to match the hash in the filename of the `.glob` file.
-Amend `pkg/arvo/app/landscape/index.html` to import the hashed JS bundle, instead
-of the unversioned index.js. Do not commit the produced `index.js` and
-make sure it doesn't end up in your pills (they should be less than 10MB each).
-
-### Tag the resulting commit
-
-What you should do here depends on the type of release being made.
-
-First, for Urbit OS releases:
+feature branch ---->   develop ---->  release ---------> master
+                          |              |                 |
+                    deployed to    deployed to         deployed to
+                    ~binnec moon ~marnec/~doznec moon  network
+```
+
+If an issue arises in the course of testing the `release` branch (because more
+people are using `marnec` than `binnec`), a PR can be opened to target
+`release`. If that's the case, the `master` needs to be merged back into
+`develop` after `release` merges into `master` to ensure that `develop` gets the
+fix.
```

Comment:
We could just always do this, for consistency sake. If they haven't diverged it would just be a no-op right? If not, maybe we can just automate the check/merge?

Notably this means develop may be "behind" release at certain points in time. Feels vaguely dangerous. I think the previous flow of always deploying to binnec first, and pulling what's there directly onto marnec when it's considered good avoids this, at the cost of keeping binnec "occupied" until a release has gone out. I'm assuming that's why we're doing the above instead?